Maquetacion/index.html

```diff
@@ -91,5 +91,10 @@ <h2 class="text title-card">Django</h2>
         </div>
       </div>
     </div>
+    <footer class="footer-bar">
+      <p class="text-f">we are game changers</p>
+      <img class="logo-w" src="assets/wolox_logo_white.svg" alt="Wolox-white">
+      <p class="copy"><small>Copyright 2018. Wolox. Allrights reserved</small></p>
```

Comment:
Beware of the text spacing: `All rights reserved`.

Reply:
👍